staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -374,10 +374,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 		shortNames = shortNamesProvider.ShortNames()
 	}
 
-	tableProvider, ok := storage.(rest.TableConvertor)
-	if !ok {
-		tableProvider = rest.DefaultTableConvertor
```

Comment:
You're allowed not to support server side get - this change returns a 406 if that happens.  This ensures that swagger matches what the storage supports.